Command/MaintenanceCommand.php

```diff
@@ -2,19 +2,20 @@
 
 namespace Northern\MaintenanceModeBundle\Command;
 
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Filesystem\Filesystem;
 
+#[AsCommand(
```

Comment:
I think we're going to have to update composer.json as well in terms of which php versions are compatible and which versions of symfony this is compatible with.